storage/sql_hooks_test.go

```diff
@@ -51,7 +51,7 @@ func TestInitAndGetSQLDriverWithLogsDriverNotFound(t *testing.T) {
 	logger := log.New(os.Stdout, "[sql]", log.LstdFlags)
 	const nonExistingDriver = "asdfaserxcvzasfd"
 	_, err := storage.InitAndGetSQLDriverWithLogs(nonExistingDriver, logger)
-	if err == nil || err.Error() != fmt.Sprintf("driver %v is not supported", nonExistingDriver) {
+	if err == nil || err.Error() != fmt.Sprintf("driver '%v' is not supported", nonExistingDriver) {
```

Comment:
You are not using the `const` here. Maybe different package?

Reply:
yes, it is in tests